tests/torchtune/training/test_activation_offloading.py

```diff
@@ -10,6 +10,8 @@
 from torch import nn
 from torchtune.training import OffloadActivations
 
+NUM_GPU_CYCLES_IN_ONE_SEC = 2000000000  # 2e9 is ~1s worth of GPU cycles
```

Comment:
```suggestion
NUM_GPU_CYCLES_IN_ONE_SEC = 2_000_000_000  # 2e9 is ~1s worth of GPU cycles
```

Reply:
Yes, I ran the repro script to confirm that memory usage + correctness were as expected. We also have a super small unit test to ensure correctness. I did not run a recipe though.